test/unit/publishing/publish-by-email.test.js

```diff
@@ -38,9 +38,9 @@ describe('Publish by email', () => {
     expect(mockNotifyClient().prepareUpload).toHaveBeenCalledTimes(1)
```

Comment:
Need to update the file `test/mocks/objects/notify-link-to-file.js` with the output of what having this enabled produces

Reply:
Adding third party component details to unit test is more like unit-testing the third party component, but for easy maintenance and readability, I have updated the file.

Reply:
It's less for unit tests and more for proper integration/system testing when we need to mock what Notify will send us to ensure our service behaves as it should.